.mailmap

```diff
@@ -1,2 +1,4 @@
-Stephen J Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
 Aaron Lehmann <aaronl@vitelus.com> <aaron.lehmann@docker.com>
+Derek McGowan <derek@mcg.dev> <derek@mcgstyle.net>
+Stephen J Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
+zhouhaibing <zhouhaibing089@gmail.com>
```

Comment:
Not a blocker, but in case @zhouhaibing089 is around to provide their real name (looks like this came in from https://github.com/docker/distribution/pull/2435)

Reply:
That is a pretty common way to write names, the numbers just shouldn't be there in the author list. If he prefers "Haibing Zhou" like on his Twitter, he is welcome to change it.

Reply:
Ah, `Haibing Zhou` is preferred.

Reply:
Just in time :)